### DISCUSSION TITLE
ant5 文本组件添加图标样式异常

### DISCUSSION THREAD

**Discussion by fj810625** [ant-design/ant-design#40636] [Q&A]
![image](https://user-images.githubusercontent.com/16619702/217493446-2ec54606-29ce-4ca0-aaa1-04e6bd2e09e1.png)

**Comment by fj810625** [comment]
大神们帮看一下这个问题如何解决

**Reply by Wxh16144** [reply]
可以使用 [antd@5.x](https://u.ant.design/codesandbox-repro) 模板创建一个最小复现，方便大家帮助你排查问题。
